index.js

```diff
@@ -0,0 +1,88 @@
+/* Set Date to Current Date */
+Date.prototype.toDateInputValue = (function() {
+  var local = new Date(this);
+  local.setMinutes(this.getMinutes() - this.getTimezoneOffset());
+  return local.toJSON().slice(0,10);
+});
+
+document.getElementById('implementationDate').value = new Date().toDateInputValue();
+
+/* Contact Form Validation */
+
+const getForm = document.querySelector('.contact-form');
```

Comment:
I usually save `get` names for functions. This is kind of personal preference though.
I alos usually match . the variable name with the class name where I can so it makes it super clear which element it is.
```suggestion
const contactForm = document.querySelector('.contact-form');
```

Reply:
Actioned in new branch